scripts/index.js

```diff
@@ -88,4 +88,24 @@ console.log(whoIsTarg(characters));
 
 // Create a histogram of the houses (it's the "allegiances" key)
 
+function housesMaster(chars) {
+    let histo = {}
+    let houses = []
+    let allegiances = chars.map(function (each) {
+        return each.allegiances;
+        })
+
+    // return allegiances;
+
+    allegiances.forEach(function (ea) {
+        if (histo.hasOwnProperty(ea)) {
```

Comment:
Nice. `hasOwnProperty` is also a solid way to do it, and is considered the correct approach when your custom key names might collide with built-in ones.